modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -421,10 +421,13 @@ protected function register_controls() {
 				'name' => 'tabs_title_background_color',
 				'types' => [ 'classic', 'gradient' ],
 				'exclude' => [ 'image' ],
-				'selector' => "{$nested_tabs_heading_selector_class} .e-n-tab-title",
+				'selector' => "{$nested_tabs_heading_selector_class} > .e-n-tab-title",
 				'fields_options' => [
 					'color' => [
 						'label' => esc_html__( 'Background Color', 'elementor' ),
+						'selectors' => [
+							'{{WRAPPER}}' => '--n-tabs-title-background-color: {{VALUE}}',
```

Comment:
I am curious about the logic.

We use a css variable here, but for the hover and active state we use a php selector.

What is the logic?

Reply:
The background color should be weaker (less explicit) than the `active` and `hover` states but stronger (more explicit) that the normal background color.
In this way, I achieve exactly that :)

Reply:
We use a css variable here, but for the hover and active state we use a php selector.

Couldn't we use the same approach everywhere?

Reply:
I am afraid we can't, if you remember, yesterday, we tried using the PHP approach in normal background color but it didn't work.
The use of the CSS variable is because we don't have a global variable in the light gray color.
This way the tab's normal background color is the BG fallback and if a color is selected that we assign the variable that appears in the PHP